packages/headless/src/features/instant-results/instant-results-actions.ts

```diff
@@ -73,7 +74,7 @@ export interface FetchInstantResultsActionCreatorPayload {
    */
   maxResultsPerQuery: number;
   /**
-   * The maximum items to be stored in the instant result list for each query.
+   * Number in milliseconds that cached results will be valid for. Set to 0 so that results never expire.
```

Comment:
> Set to 0 so that results never expire.

Shouldn't it be something like: `Set to 0 so that results are never cached.` ?

Reply:
This comment correctly reflects the current behavior though: a value of 0 is interpreted as "never expire".

See: https://github.com/coveo/ui-kit/blob/3519baf5857089d7c91f4ff32002cb0885382d69/packages/headless/src/features/instant-results/instant-results-state.ts#L30-L35

I believe the reason for this behavior is because we don't want to encourage customers to have "no cache", since we expect `updateQuery` to be called often (e.g., whenever somebody hovers a search suggestion). Whenever `updateQuery` is called, a whole new `/v2` search query is triggered if the cache is expired.

Reply:
Alright then omit my comment, this seems completely unrelated to our SearchAPI cache.Thanks for the explanation!